configure.cmd

```diff
@@ -221,7 +221,7 @@ if "%VS_SOLUTION%" == "1" (
     set REACTOS_OUTPUT_PATH=%REACTOS_OUTPUT_PATH%-sln
 )
 
-if "%REACTOS_SOURCE_DIR%" == "%CD%\" (
+if /I "%REACTOS_SOURCE_DIR%" == "%CD%\" (
```

Comment:
From what I see, this line is the only change in the PR that's actually needed? If so, I'd advise removing the other changes.